src/libraries/System.Net.Security/src/System/Net/Security/Pal.OSX/SafeDeleteSslContext.cs

```diff
@@ -225,7 +225,17 @@ internal void Write(byte[] buf, int offset, int count)
                     _fromConnection.Enqueue(buf[offset + i]);
                 }
             }
+        }
```

Comment:
The existing array-based method can just use the span-based one now:
```C#
Write(buf.AsSpan(offset, count));
```
to avoid the duplication.  Or the array-based call site could just use AsSpan and the above function could be deleted entirely.